VK_V1.0/VK_V1.0/Animations/NavigationAnimations/CustomNavigationController.swift

```diff
@@ -0,0 +1,54 @@
+// CustomNavigationController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+///  Кастомный контроллер навигации
+final class CustomNavigationController: UINavigationController, UINavigationControllerDelegate {
```

Comment:
```suggestion
final class MainNavigationController: UINavigationController, UINavigationControllerDelegate {
```
Custom не употребляем